src/vs/workbench/browser/parts/views/viewsService.ts

```diff
@@ -367,13 +367,16 @@ export class ViewsService extends Disposable implements IViewsService {
 					const viewsService = serviceAccessor.get(IViewsService);
 					const viewContainerLocation = viewDescriptorService.getViewContainerLocation(viewContainer);
 					switch (viewContainerLocation) {
-						case ViewContainerLocation.Sidebar:
-							if (!viewsService.isViewContainerVisible(viewContainer.id) || !layoutService.hasFocus(Parts.SIDEBAR_PART)) {
+						case ViewContainerLocation.AuxiliaryBar:
```

Comment:
Thanks for this change @connor4312 ... Looks like this command got forgotten in the changes with the views. Your fix actually highlights another problem with this command. The title of the command is Show X or Toggle X depending on where the view container lives *at startup*. The behavior however is based on the location at execution. I'm not really sure if there is a strong reason for having this differentation so I suggest we stick to the same behavior for all locations: Show X and simplify here. @sandy081 what do you think?

Reply:
@sbatten This was brought multiple times and got dropped. See the discussion here https://github.com/microsoft/vscode/issues/7540. I agree the change would be simple and consistent but not might be acceptable by all users who got used to it.

Reply:
If we don't go for consistency, we need to fix the bug where the command name doesn't reflect the behavior. And we also need to make a decision on whether the Secondary Side Bar acts more like the panel or more like the Primary Side Bar. I would say Side Bar behavior, since that's what its called and that would be my vote for all view "commands".